### ISSUE TITLE
Can't see the ground texture

### ISSUE THREAD

**Issue by m0ose** [CodeArtemis/TriggerRally#7]
I cant see the ground. The trees, and signs and car show up, but not the ground

**Comment by jareiko** [comment]
Thanks for the report! What browser and version are you using?

**Comment by Jasz** [comment]
I'm having the same problem. I'm using Firefox 18 (Aurora).

**Comment by jareiko** [comment]
I can reproduce this in Firefox 18 beta:

URL: http://triggerrally.com/x/Alpina/ArbusuG/drive

```
[10:35:47.033] Error: WebGL: DrawElements: bound vertex attribute buffers do not have sufficient size for given indices from the bound element array @ http://triggerrally.com/build-v2/drive-main.js:1437
```

I suspect it may be a Firefox bug, since it worked in previous versions and in Chrome, but I'd like to investigate further before filing a bug with them.

**Comment by Tonysp** [comment]
Sometimes it happens also to me. I have Chrome 23.0.1271.95. However, when i restart the browser, then its ok.

**Comment by jareiko** [comment]
I reduced this to a small test case that seems to show a Firefox WebGL bug:

https://bugzilla.mozilla.org/show_bug.cgi?id=825205

**Comment by ddorstijn** [comment]
I can't see the ground too. I'm just using Chrome on a Windows xp.

**Comment by bjacob** [comment]
Thanks for the report. This is now fixed in Firefox nightlies, and we are trying to get approval to backport the fix to Firefox 18 and 19.

**Comment by bjacob** [comment]
It is too late to fix this in Firefox 18. This might be fixed in a 18.0.1, but in any case, there will be a time window during which the released Firefox will have the regression.

As a work-around you could round up your vertex buffers that are greater than 64k \* (vertex size) to the next multiple of 64k \* (vertex size).

**Comment by jareiko** [comment]
Thanks for fixing, and for the suggested workaround! Sorry I didn't get the bug report to you sooner.

**Comment by jareiko** [comment]
Confirmed fixed as of Firefox 19.0 beta.
